gobblin-runtime/src/main/java/org/apache/gobblin/runtime/task/FailedTask.java

```diff
@@ -16,19 +16,30 @@
  */
 package org.apache.gobblin.runtime.task;
 
+import org.apache.gobblin.configuration.ConfigurationKeys;
 import org.apache.gobblin.configuration.WorkUnitState;
 import org.apache.gobblin.publisher.DataPublisher;
 import org.apache.gobblin.publisher.NoopPublisher;
 import org.apache.gobblin.runtime.JobState;
 import org.apache.gobblin.runtime.TaskContext;
+import org.apache.gobblin.runtime.TaskState;
 import org.apache.gobblin.source.workunit.WorkUnit;
 
+import com.google.common.base.Throwables;
+
+import groovy.util.logging.Slf4j;
+
+
 /**
- * A task which raise an exception when run
+ * A task which returns "FAILED" state directly.
```

Comment:
If the code on the left raises an exception, why the code on the right would not raise an exception? javadoc on the left is incorrect?